easybuild/easyconfigs/g/GenomeTools/GenomeTools-1.5.10-foss-2018b.eb

```diff
@@ -0,0 +1,34 @@
+easyblock = 'ConfigureMake'
+
+name = 'GenomeTools'
+version = "1.5.10"
+
+homepage = 'http://genometools.org'
+description = """A comprehensive software library for efficient processing of structured genome
+  annotations"""
+
+toolchain = {'name': 'foss', 'version': '2018b'}
+
+source_urls = ['https://github.com/genometools/genometools/archive']
+sources = ['v%(version)s.tar.gz']
+checksums = ['a6aa7f158a3cef90fea8d0fe24bfad0c3ee96b17b3ba0c1f6462582593af679e']
+
+dependencies = [
+    ('cairo', '1.14.12'),
+    ('Pango', '1.40.14'),
```

Comment:
Is there a reason you're not using the existing Pango 1.42.4 we already have for foss/2018b?

Reply:
@akesandgren Most likely that's just an oversight, we can take over the reigns on this one by bumping this to 1.42.4 and getting rid of the `Pango` easyconfig being added here?